src/global/global_helpers.R

```diff
@@ -3846,6 +3845,10 @@ convert_to_gl <- function(x, input_unit, molecule) {
         return(x)
     }
 
+    if(grepl('ppm', input_unit)){
```

Comment:
derelict chunk?

Reply:
@vlahm Great catch, thanks! I was going to build that but found it not necessary.